### DISCUSSION TITLE
Windows 10 32Bit, any modules to perform some cleanups

### DISCUSSION THREAD

**Discussion by toxsickcity** [Atlas-OS/Atlas#1223] [Help & Support]
Hi I have a very low performance tablet and I would like to remove some bloat.
Is there an older atlasOS for 32bit windows?
or can I grab some powershells to remove edge and other crap?

cheers

**Comment by Ast3risk-ops** [comment]
1. no
2. https://github.com/he3als/edgeremover
3. Just get a Linux distro instead of using Windows on a machine that old,
